src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -2629,6 +2631,10 @@ public static class PKAnalysis
          public static readonly string Unit = UI.Unit;
          public static readonly string Description = "Description";
          public static readonly string Warning = "Warning";
+         public static readonly string OnCurves = "On Curves";
```

Comment:
@abdelr @PavelBal This constants were defined as per request...but not USED! So the hard coded strings were still used in the code behing